metaflow/plugins/aws/batch/batch.py

```diff
@@ -145,7 +145,6 @@ def launch_job(
             .environment_variable('METAFLOW_USER', attrs['metaflow.user']) \
             .environment_variable('METAFLOW_SERVICE_URL', BATCH_METADATA_SERVICE_URL) \
             .environment_variable('METAFLOW_SERVICE_HEADERS', json.dumps(BATCH_METADATA_SERVICE_HEADERS)) \
-            .environment_variable('METAFLOW_DATASTORE_SYSROOT_LOCAL', DATASTORE_LOCAL_DIR) \
```

Comment:
Can you add a comment to the code outlining why `METAFLOW_DATASTORE_SYSROOT_LOCAL` isn't needed here, so that it doesn't get added inadvertently later on? cc - @romain-intel.

Reply:
Yes. I can update the PR by putting a comment on top?